desktop/client.js

```diff
@@ -15,7 +15,13 @@ http.get('http://localhost:4000/dist/main.bundle.js', function (response) {
   response.pipe(file)
   file.on('finish', function() {
     file.close(function () {
-      const e = spawn(electron, [name], {stdio: 'inherit'})
+      const params = [name]
+
+      if (process.env.USE_INSPECTOR) {
+        params.unshift('--debug-brk=5858')
```

Comment:
This is to debug the main electron thread (the node one)

On Friday, February 5, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In desktop/client.js
> <https://github.com/keybase/client/pull/1915#discussion_r52090066>:
>
> > @@ -15,7 +15,13 @@ http.get('http://localhost:4000/dist/main.bundle.js', function (response) {
> >    response.pipe(file)
> >    file.on('finish', function() {
> >      file.close(function () {
> > -      const e = spawn(electron, [name], {stdio: 'inherit'})
> > +      const params = [name]
> > +
> > +      if (process.env.USE_INSPECTOR) {
> > +        params.unshift('--debug-brk=5858')
>
> What does this do?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1915/files#r52090066>.
>